[FIX] web, website: fix some details about the "Card" snippet redesign

sobo-odoo opened the pull request:
[FIX] web, website: fix some details about the "Card" snippet redesign

In commit [1], the `s_card` snippet has been redesigned and new options
have been added for it.

This commit fixes some details and behaviors about it:
- remove the need of a `001.scss` file and therefore also remove the
unneeded `data-vcss` attribute (also in the snippets using the card).
- rename the `001.xml` file by `options.xml`, as it is more correct
since its content is related to the options.
- move the "Alignment" option in a better place in the right panel, hide
it when not necessary (i.e. when the card width is at 100%) and display
a default value.

[1]: https://github.com/odoo/odoo/commit/dd40e1e8d9026d189dc17134c85331711f50a470

Related to task-3674888

---
[FIX] website: keep the "Timeline" snippet cards aligned in mobile view

In commit [2], the "Timeline" snippet has been redesigned. When the
"Alignment" option is used, it allows to position the card closer or
further from the line. In mobile view, the line is not in the center
anymore but on the left. However, the cards keep the alignment they had
in desktop view, which makes them misaligned in mobile.

This commit fixes this misalignment by forcing these cards to be aligned
on the left in mobile view.

[2]: https://github.com/odoo/odoo/commit/48545a60ddc4a50bc792318ed6cd20cfb20b5c55

Related to task-3657763

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/178075.png)](https://mergebot.odoo.com/odoo/odoo/pull/178075)

sobo-odoo commented:
@stefanorigano FYI, here is the PR fixing some card issues (discussed with QSM) :slightly_smiling_face: 
Note that I took a look at the "[Three columns PR](https://github.com/odoo/odoo/pull/176684)" and I saw that you fixed the Alignment option, but some things were not correct:
- you declared it in the `CardImageOptions` while I think it should be in the `CardWidth` one (since it is even one of its sub-option, given the sublevel class) :wink: 
- a default value for it was missing (already the case before)
- you kept some dependencies that were not needed anymore (`so_width_100`)

Therefore, I fixed that directly here in my PR, since I was asked to fix that too :+1: But note that I added you as co-author, since I took a part of the code you made :slightly_smiling_face:

stefanorigano commented:
Hei @sobo-odoo , thanks for the heads-up!
I'll keep an eye on this PR and rebase [mine](https://github.com/odoo/odoo/pull/176684) accordingly when the time will come 👍

>  But note that I added you as co-author,

It wasn't not necessary, it's just 2 lines... by I greatly appreciate the gesture 🙂

qsm-odoo commented on addons/web/static/src/scss/bootstrap_review_frontend.scss:
```diff
@@ -16,7 +16,7 @@
 
 // Cards
-:where(.card:not([data-vcss])) .card-body {
+:where(.card:not([data-vxml])) .card-body {
```
To whoever sees this: as discussed, we'll get rid of this rule somehow, before the release, I have to think about it :+1:

qsm-odoo reviewed (approved):
@robodoo rebase-ff r+

robodoo commented:
Merge method set to rebase and fast-forward.
